framework_tvb/tvb/core/adapters/constants.py

```diff
@@ -73,3 +73,8 @@
 
 ALL_TYPES = (TYPE_STR, TYPE_FLOAT, TYPE_INT, TYPE_UPLOAD, TYPE_BOOL, TYPE_DICT,
              TYPE_ARRAY, TYPE_SELECT, TYPE_MULTIPLE, TYPE_DYNAMIC, TYPE_LIST)
+
```

Comment:
A rebase with master would be nice to have, as I recently cleaned `constants.py`